ArcGISRuntimeSDKQt_CppSamples/Layers/RasterLayerFile/RasterLayerFile.h

```diff
@@ -39,7 +39,7 @@ class RasterLayerFile : public QQuickItem
   void componentComplete() override;
   static void init();
 
-  Q_INVOKABLE void createAndAddRasterLayer(QUrl rasterUrl);
+  Q_INVOKABLE void createAndAddRasterLayer(QString dataPath);
```

Comment:
`const QString&`

Reply:
I actually end up modifying the path in the implementation of this method